CODEOWNERS

```diff
@@ -1 +1,9 @@
-*   @davidpmccormick @gestchild @jamesgorrie
+# These are default codeowners, later rules take precedence
+*	@wellcomecollection/js-ts-reviewers
```

Comment:
I've created a new team with basically the same set of reviewers to manage this a bit more easily: @wellcomecollection/js-ts-reviewers